src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -0,0 +1,53 @@
+package org.testeditor.fixture.host;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testeditor.fixture.host.net.Connection;
+import org.testeditor.fixture.host.s3270.options.CharacterSet;
+import org.testeditor.fixture.host.s3270.options.TerminalMode;
+import org.testeditor.fixture.host.s3270.options.TerminalType;
+
+public class HostDriverFixture {
```

Comment:
We should use Workspace Mechanics for harmonizing our code for the whole team members.